.github/workflows/publish.yml

```diff
@@ -39,4 +39,62 @@ jobs:
           DWDS_HTTP_BASIC_ACCESS_AUTHENTICATION: ${{ secrets.DWDS_HTTP_BASIC_ACCESS_AUTHENTICATION }}
         run: |
           cd kiwix-android
-          eval "./gradlew publish${TAG^}ReleaseApkWithExpansionFile"
+          eval "./gradlew publish${TAG^}ReleaseBundleWithPlayAssetDelivery"
+
+  publish_dummy_bundle:
+    runs-on: ubuntu-22.04
+
+    steps:
+      - name: Retrieving custom app configuration
+        uses: actions/checkout@v3
+
+      - name: Retrieving Kiwix Android source code
+        run: git clone --depth=1 --single-branch --branch develop https://github.com/kiwix/kiwix-android.git
+
+      - name: Copying custom app configuration into Kiwix Android code base
```

Comment:
@MohitMaliDeveloper Should we add this tag here? Have we tested this if it works?